07-task-aliasing.Rmd

```diff
@@ -0,0 +1,80 @@
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+# Calling the same task more than once 
+It is not uncommon that the same task (for example the BwaMem task in our example) in a workflow is required to be used multiple times. For example you may want to do this if you need to run it using a different set of inputs, if you want to split your task to scale your workflow and optimize compute resources. 
```

Comment:
Change for clarity and connection to chapter 6:

"We've already gone over running a task multiple times in the context of scattered tasks. However, you may also want a task to run more than one time if that task is to run on multiple sets of inputs. In our case, we want to run a similar analysis on tumor samples and samples taken from normal tissue."